rsconf/component/network.py

```diff
@@ -100,6 +100,7 @@ def internal_build_write(self):
         jc = self.j2_ctx
         jc = self.j2_ctx
         z = jc.network
+        self._restricted_public_tcp_ports(z)
```

Comment:
I think it is worth commenting this bit of code that it is important it is the last thing to mutate the lists of ports.

Reply:
c24d8fa added comments to iptables as well.